code/main/services/networkService.brs

```diff
@@ -86,11 +86,11 @@ function _adb_NetworkService() as object
                         responseCode = msg.GetResponseCode()
                         responseString = msg.getString()
                         failureMessage = msg.GetFailureReason()
-                        _adb_logDebug("syncPostRequest() -  Received response code:(" + FormatJson(responseCode) + ") body:(" + FormatJson(responseString) + ") message:(" + FormatJson(failureMessage) + ").")
+                        _adb_logDebug("syncPostRequest() -  Received response code:(" + FormatJson(responseCode) + ") body:("  + chr(10) + responseString  + chr(10) + ") message:("  + chr(10) + failureMessage  + chr(10) + ").")
```

Comment:
Why not use `\n` instead of chr(10)?

Reply:
Let me try that. I thought that doesn't work.

Reply:
\n doesn't work.